\chapter{Results}    
\label{Ch:results}

\section{Rvs localization in yeast endocytosis}

Averaged centroid tracking, as described in Picco et al., can quantify the movement and dynamics of endocytic proteins along the timeline of membrane invagination and scission. Briefly described, yeast cells expressing fluorescently-tagged endocytic proteins are imaged at the equatorial plane. Since membrane invagination progresses perpendicularly to the plane of the plasma membrane, proteins that move inward with membrane ingression now do so in the imaging plane. Centroids of many endocytic patches, for example, containing Sla1-GFP, are thus tracked. Since yeast endocytic sites show highly stereotypic behavior, the centroids of the proteins involved can be averaged as they arrive at and depart from the plasma membrane, leading to an averaged centroid tracked with high spatial and temporal resolution. When different endocytic proteins are simultaneously imaged with the abundant actin binding protein Abp1, Abp1 provides a frame of reference to which the other proteins can be now aligned. 

	\begin{figure}
	\centering
	\includegraphics[width=19.5cm,height=19.5cm,keepaspectratio, valign=t]{figures/results_final/yeast_schemat_fig1_C}
	\caption[Centroid tracking yeast endocytic proteins]
	{1A: Above: Schematic of a yeast cell, showing the equatorial plane. Below: Cross section of the cell at the equatorial plane, with fluorescently tagged endocytic proteins at the plasma membrane. 
	B: Kymographs of Sla1-GFP and Rvs167-GFP at endocytic patches at the plasma membrane. Movement of coat protein Sla1-GFP shows slow inward movement, while Rvs167-GFP shows a sharp jump into the cytoplasm that is concomitant with membrane scission and vesicle formation. Exposure rate for kymograph, 80ms.
	C: Schematic of the timeline of membrane invagination during endocytosis, with Sla1, Abp1 and Rvs167 indicated. 
	D, E: Movement and normalized fluorescent intensity of Sla1, Abp1 and Rvs167 during endocytosis. Sla1 follows the membrane as it moves  into the cytoplasm. Rvs arrives at membrane tubes, potentially formaing a scaffold. At scission, the scaffold along the tube is disassembled, resulting in an inward jump of the Rvs167 centroid, to protein localized at the base of the newly formed vesicle, and a sharp decay in fluorescent intensity. Abp1 intensity drops after membrane scission. D and E are aligned in so that time=0 (sec) corresponds to scission time.
	\label{fig1_schematic}}
	\end{figure}

\mbox{}\\
Centroid tracking shows that the late-stage coat protein Sla1 arrives and accumulates at endocytic sites, starts to moves into the cytoplasm at the onset of actin polymerization 1,2. Sla1 moves into the cytoplasm along with the membrane and follows its movement during endocytosis. In WT cells, Sla1 moves about 140nm into the cytoplasm, at which point vesicle scission occurs. It will be used throughout this work as the marker for membrane movement. As inward movement of the membrane begins, the Sla1 coat is disassembled, inferred from the decay of the fluorescent intensity of Sla1-GFP 2, shown in \ref{fig1_schematic}. As has been shown before, Rvs localizes to endocytic patches at the yeast plasma membrane in the late scission-stage1,2. Rvs167 arrives after a parallel membrane tube is formed, and scission occurs at 60\% of its lifetime at the plasma membrane1,3. At the time of scission, the Rvs167-GFP centroid shows a sharp jump into the cytoplasm, a profile that is unique among endocytic proteins. Concomitantly, fluorescent intensity of Rvs167-GFP shows a sudden decay. Abp1 is used throughout this work as a marker for the actin network. Abp1 intensity also peaks at scission time, and drops after scission, indicating disassembly of the actin network upon vesicle formation. Thus, in WT cells, fluorescence intensity of Abp1 and Rvs167 begin to drop at the same time. Henceforth, unless indicated, "scission time", or time=0 (s) refers to the time at which fluorescent intensity of averaged Abp1 patches peaks. The other endocytic proteins are aligned in this timeline as described by Picco et al., by simultaneous dual-color imaging of the respective proteins with Abp1. The movement away from the plasma membrane of the averaged centroid of a protein is referred to as "movement". Averaged centroids for different GFP-tagged proteins are obtained from tracking 30-50 endocytic patches in 10-15 yeast cells. 



\section{Rvs deletion phenotype}
\label{sec:rvsdel}

The Rvs complex, as has been discussed in section \ref{Ch:Intro}, influences efficiency of membrane scission. Recruitment in the final stage of membrane movement, localization to the membrane tube, and  disassembly concomitant with scission all indicate that Rvs could mechanistically influence the scission process. In order to quantify this influence, I tracked Sla1-GFP in \textit{rvs167$\Delta$} cells and compared its movement against WT Sla1-GFP. Rvs167 and Rvs161 are obligate heterodimers at endocytic sites, and deletion of one of them is expected to result in the loss of the other during endocytosis. Movement of Sla1 is reduced in \textit{rvs167$\Delta$} to about 60nm instead of the 140nm seen in WT cells. CLEM studies have shown that Rvs167 only localizes to endocytic sites after the tubes are 60nm long: membrane invagination is unaffected in \textit{rvs167$\Delta$} cells till the expected time of arrival of Rvs, after which membrane scission occurs, rather than further invagination. This indicates that first, membrane scission can occur at invagination lengths of 60nm. Then, that the arrival of Rvs prevents membrane scission at this point and allows further membrane invagination. 

\begin{figure}[h]
	\centering
	\includegraphics[width=14cm,height=14cm,keepaspectratio, valign=t]{figures/results_final/rvsdeletion}
	\caption[some stuff]
	{A: Averaged centroid movement of Sla1-GFP in WT and \textit{rvs167$\Delta$} cells. Time=0 (s) for WT Sla1 is scission time. rvs167$\Delta$ Sla1 is shifted to move at the same time as WT B: Vesicle surface area for WT and \textit{rvs167$\Delta$} determined by CLEM. Data used with permission from Wanda Kukulski.
	\label{fig2_rvsdelta}
	}
\end{figure}



\section{Recruitment of Rvs and function of domains} 

	\subparagraph{BAR domains: curvature sensing or generating? }
	Cellular membrane shape is a result of properties like rigidity, tension, intracellular pressure, that are all influenced by membrane lipid composition and the proteins embedded in it1,2. Since tension, pressure, and rigidity all oppose membrane deformation, energy is required to deform and bend it. BAR domains can generate curvature if the energy required to deform the membrane is less than the energy spent in binding flat membrane.

	\vspace{5mm}
			
	Curvature-generation by scaffolding and imposing its own shape on membranes has been extended to various types of BAR proteins, (Arkhipov et al., 2009; Frost et al., 2008; Henne et al., 2007; Itoh et al., 2005; Pykalainen et al., 2011; Saarikangas et al., 2009; Shimada et al., 2007; Yu and Schulten, 2013). In order for BAR scaffolds to impose membrane curvature, some requirement have to be met3: they have to have present a large membrane-interacting surface that can mediate membrane binding, have intrinsic curvature that can be imposed on the surface, and have a rigid structure that can overcome bending resistance of the membrane. Because of their shape (Peter 2004, Gallop 2006, Weissenhorn 2005), and their capacity to oligomerize into large assemblies on tubes (Mim 2012, Mizuno 2010, Takei 1999, Yin 2009), it has been suggested that BAR domains impose their shape on the membrane, and generate membrane curvature on cellular membranes. Further, it has been shown that the central BAR region is rigid and required for tubulation, both in-vivo and of liposomes4. The N-helix of NBAR domains can also generate curvature independently of the BAR scaffold (Varkey 2010, Westphal and Chandra 2013). In endophilin, the BAR domain is relatively far from the membrane, suggesting a mechanism dependent on the N-helix (Jao 2010). Different BAR domains thus likely employ different mechanisms to interact with the membrane for generating vesicles, and tubes (Ambroso 2014). For example, the N-helix of endophilin is necessary for liposome binding5, while that of amphiphysin is important, but not necessary6. 


	\vspace{5mm}
	Curved BAR proteins that can induce curvature are also able to sense curvature: in-vitro, BAR domains show a preferential-binding to vesicles based on their intrinsic curvature. Curvature-generation and sensing seem to intrinsically coupled mechanisms. That BAR domains are able to generate curvature does not imply that this is their function, at least in endocytosis: in-vivo, the significance of curvature-generation is not determined. Tracking over thirty different endocytic proteins in NIH-3TC cells (derived from mouse fibroblasts), TIRF imaging shows that Endophilin2 and Amphiphysin1 arrive late in the endocytic time-line right before scission7, suggesting they arrive when membrane tubes are already formed. 


	\vspace{5mm}
	In the case of Rvs, as has been shown earlier, and seen in \ref{sec:rvsdel}, the Rvs complex localizes to sites late in the endocytic timeline, close to scission8. CLEM studies have further shown that Rvs localizes to sites after the membrane invaginations are about 60nm deep into the cytoplasm: Rvs localizes once membrane curvature is established. Whether this localization is dependent on membrane curvature, recognized by the BAR domain has not been shown. 

	\subsection{BAR domain senses membrane curvature in-vivo}
	To test whether Rvs is recruited because of membrane curvature, I first imaged Rvs167-GFP without the BAR domain, that is Rvs167-delsh3-GFP (henceforth referred to as BAR). BAR-GFP forms cortical patches \ref{fig2_sla2del}, so the BAR domain is able to localize to the plasma membrane in the absence of the SH3 domain. In a yeast strain expressing both BAR-GFP and Abp1-mCherry, BAR-GFP co-localizes with Abp1, indicating that BAR domains are recruited to endocytic patches \ref{fig2_sla2del}). In order to test whether this localization is due of membrane curvature, I compared the dynamics of Rvs167-GFP against BAR-GFP in \textit{sla2$\Delta$} cells \ref{fig2_sla2del} (D-F). Sla2 is a coat protein that acts as a linker between the membrane and the actin cytoskeleton by binding both via its N-terminal ANTH domain, and its C-terminal THATCH domain. This allows forces generated within the actin network to be transmitted to the membrane11. In \textit{sla2$\Delta$} cells, rather than cortical actin patches, an “uncoupling phenotype” is observed11,12. Although endocytic coats are formed, actin is polymerized continuously at these sites, the membrane is not pulled inwards, and vesicles are not formed: forces generated by the actin network are not transmitted to the membrane (\ref{fig2_sla2del} D-F).

	\begin{figure}
	\centering
	\includegraphics[width=21cm,height=21cm,keepaspectratio]{figures/results_final/sla2_del_final6}
	\caption [Localization of Rvs167 and BAR with and without membrane curvature]
	{A: Max. intensity projections of time-lapse images of cells expressing GFP-tagged full length (FL) Rvs167 or BAR with Abp1-mCherry. Exposure rate 250ms. B: Schematic of endocytic progression in WT and BAR cells (see section \ref{delsh3_movement}).
	C: Montage of Rvs167-GFP and Abp1-mCherry at the plasma membrane in WT and BAR cells . Each frame of montage is every third frame of time-lapse images from A. 
	D: Max. intensity projection of time-lapse images of \textit{sla2$\Delta$} cells expressing GFP-tagged FL or BAR Rvs167 and Abp1-mCherry. 
	E: Schematic of membrane invagination in \textit{sla2$\Delta$} cells. F: Montage of GFP tagged FL and BAR Rvs167, and Abp1-mCherry. Exposure rate 1000ms for GFP, 800ms for RFP. 
	G: Max. intensity projection of time-lapse images of \textit{sla2$\Delta$} cells of GFP-FL or BAR Rvs167 and Abp1-mCherry, after treatment with LatA for 10’. Exposure rate 1000ms for GFP, 800ms for RFP. H: Schematic of membrane invagination in \textit{sla2$\Delta$} cells treated with LatA. 
	All scale bars = 2um.
	\label{fig2_sla2del}}
	\end{figure}

	\vspace{5mm}
	In \textit{sla2$\Delta$} cells, Rvs167-GFP is recruited to the plasma membrane \ref{fig2_sla2del} at the plasma membrane, and together with Abp1-mCherry. Some Rvs167-GFP patches persist at the plasma membrane, while many are assembled and disassembled at Abp1 patches. Some Rvs167 patches do not co-localize with Abp1. In \textit{sla2$\Delta$} cells expressing BAR-GFP, localization is mostly removed except for rare transient patches at the plasma membrane that are co-localized with Abp1, while most of the patches appear to be recruited independent of Abp1. Rvs167-GFP and BAR-GFP patches are both dynamic, indicating an interaction exists in both cases that is able to assemble and disassemble Rvs patches at the plasma membrane. 

	\subsection{The SH3 domain is able to localize Rvs in an actin and curvature-independent manner}
	\label{delsh3_latA}
	As I show in the previous section, full-length Rvs is able to localize to cortical patches in \textit{sla2$\Delta$} cells. This localization must come from the SH3 domain, since BAR alone does not localize in cells without sla2. We expected that the SH3 domain must interact with WASP or actin-binding proteins: an interaction with Abp1 has been shown, as well as with Las17, type I Myosins, and Vrp1. In order to prove this, I imaged BAR-GFP and Abp1-mCherry in \textit{sla2$\Delta$} cells treated with the actin sequestering agent LatrunculinA (LatA). LatA is a sea-sponge toxin that binds monomeric actin and prevents incorporation of actin into filaments. Since high actin turnover is required at endocytic sites, LatA effectively disassembles WASP components and other actin-binding proteins of the endocytic machinery, and blocks endocytosis. In combination with the sla2 deletion, latA treatment will effectively prevent membrane curvature as well as remove actin-binding proteins from endocytic sites. Loss of actin binding proteins is verified by the loss of Abp1 signal in the RFP channel.


\newpage
Surprisingly, full-length Rvs is transiently localized to the plasma membrane in spite of the LatA treatment, suggesting that the SH3 domain is able to recruit Rvs to the plasma membrane. This recruitment occurs in the absence of a BAR-membrane interaction, since BAR-GFP localization is completely removed in LatA treated cells. Rvs167-GFP patches are transient, so an assembly-disassembly mechanism is mediated by the SH3 domain outside of its BAR domain interaction. Localization of Rvs161, which does not have an SH3 domain, is also removed by LatA treatment12, supporting the conclusion that the BAR domains of the Rvs complex senses membrane curvature in-vivo. 

	\subsection{Loss of the SH3 domain affects progression of \\endocytic sites}
	\label{delsh3_movement}
Since the Rvs167 SH3 domain plays a surprisingly large role in the function of Rvs, I investigated its effect further. SH3 domains are known to mediate protein-protein interaction by binding to proline-rich sequences that contain a core PXXP motif13,14 (where X is any amino acid). These domains are ubiquitous in cellular interaction pathways, and several endocytic proteins have at least one SH3 domain, found to self-regulate activity, as well as to modulate local concentrations of protein. Although SH3 domains are abundant, they appear to have specific of binding partners. For the Rvs167 SH3 domain, neither the specific binding partner, nor its function in the scheme of endocytosis is known. From early work, the BAR domain is expected to act as the functional module of the Rvs proteins: most of the phenotypes of Rvs deletion can be compensated by expression of the BAR domain alone, although the SH3 domain is required in addition to the BAR domain for bipolar budding pattern15. 

	\vspace{5mm}
In order to probe the contribution of the Rvs167 SH3 domain to endocytosis, I studied membrane and scission dynamics by expressing Rvs167 without the SH3 domain (that is, BAR domain). Sla1 moves inwards to a maximum of approximately 50nm instead of the 140nm found in WT invaginations. Shorter invaginations with a maximum length of 60nm have been observed in the case of Rvs167 deletion by CLEM 10, which is approximately the same length as those observed in the BAR cells: loss of the SH3 domain appears to be detrimental to the function of the Rvs complex. That reduced inward movement of the coat seen in centroid tracking reflects membrane scission at shorter invagination lengths is confirmed by CLEM on strains expressing Rvs167-GFP and Abp1-mCherry in WT and BAR strains (Fig.\ref{fig2_sh3del}E). Upon preliminary analysis, invagination lengths measured in the BAR strain are distributed around \textit{xxxxxxxxxxxxxxxxx}

	\vspace{5mm}
Quantification of the number of BAR-GFP molecules recruited to endocytic sites shows that without the SH3 domain, recruitment is reduced by half: the maximum numbers of BAR-GFP recruited is 50\% of WT, at 57 +/- 9.9 molecules, compared to 113.5 +/- 5.2, although the cytoplasmic concentration of protein is not affected compared to Rvs167-GFP (see methods). Movement of Rvs167-GFP into the cytoplasm at scission time is reduced in BAR cells, and 10.4\% of Rvs167 patches remain on the plasma membrane and are disassembled without inward movement , compared to only 2.56\% in WT.  Abp1-GFP recruitment without the SH3 domain is similarly reduced to 65\% of WT recruitment, from a maximum number of 459.5 +/- 19.9 molecules in BAR to 302.9 +/- 12.9 in WT. 

\begin{figure}
	\centering
	\includegraphics[width=21cm,height=21cm,keepaspectratio]{figures/results_final/delsh3_9}
	\caption [Effect of the Rvs167 SH3 deletion]
	{A: Movement of Sla1 and Rvs167 in WT and BAR strains. Centroids are aligned in time so that time=0 corresponds to the Abp1-mCherry fluorescent intensity peak in simultaneous dual-color imaging of the corresponding strains. 
	C: Maximum number of molecule of Abp1-GFP in WT, and BAR cells, and Rvs167-GFP in WT and BAR cells, with standard error of mean. p values from a two-sided z test, * = p $\leq$ 0.05 , ** = p$\leq$ 0.01, *** = p $\leq$ 0.001. 
	D: Difference in time between arrival of Sla1-mCherry and Rvs167-GFP, and Abp1-mCherry and Rvs167-GFP in WT and BAR strains. Exposure 560ms for each channel. Mean and standard error of the mean are shown, * = p$\leq$ 0.05, ** = p$\leq$ 0.01, *** = p$\leq$ 0.001. p values of two-sided t test.
	E: Invagination lengths in WT and BAR cells measured by CLEM in cells expressing Rvs167-GFP and Abp1-mCherry. 
	\label{fig2_sh3del}}

	\end{figure}
	\vspace{5mm}
	
For a more detailed inquiry into changes in the endocytic machinery without the Rvs167 SH3 domain, I quantified the lifetimes of Rvs, and coat and actin network using Sla1 and Abp1 as markers in BAR cells using total internal reflection fluorescence (TIRF) microscopy. Unlike epifluorescence microscopy at the equatorial plane, which has been the method used for quantification so far, when using TIRF, only fluorophores up to a depth of about 100nm from the glass-sample interphase are excited. This reduces fluorescent signal from the cytoplasm, allowing detection of low intensity fluorescent signal, and is a better method for visualizing the early incidence of a protein at the plasma membrane than epifluorescence microscopy. Arrival of Rvs167-GFP, coexpressed with either Sla1-mCherry or Abp1-mCherry in BAR and WT cells are compared. 

	
	\vspace{5mm}
While the lifetimes of Rvs167 in WT and BAR are similar, as are Sla1 lifetimes in both cell types, there is a significant increase in the lifetime of Abp1 lifetime in BAR cells (not shown). I looked for differences in the timing of recruitment of endocytic proteins by looking at the difference in time between recruitment of Sla1 and Rvs167, and the difference in time between recruitment of Abp1 and Rvs in the WT and BAR cells. The time difference between recruitment of Sla1 and Rvs167 is unchanged between WT and BAR cells, while the difference in time between recruitment of Abp1 and Rvs167 is increased in BAR cells (Fig.\ref{fig3_sh3del}D).


		
\section{Function of Rvs}

	\subsection{BAR domains as scaffold for dynamin}
	\subparagraph{Yeast dynamin Vps1}
	
	Yeast dynamin is the obvious solution to membrane scission. Although none of the three dynamin- like proteins has a proline-rich domain, one of the yeast dynamins, Vps1 has been suggested to be involved in endocytosis11,12. Rooij et al., suggest that Vps1 localizes to endocytic sites in the late scission stage, and that the \textit{vps1Δ rvs167Δ} double mutant increases membrane retraction rates after invagination, an indication of scission failure. Vps1-GFP does not localize to endocytic sites in Gadila et at.,13, but localizes to the golgi body and to vacuoles. Kishimoto et al, do not find a colocalization between Vps1 and Abp1 localization, and also report that the \textit{vps1Δ rvs167Δ}  double mutation does not affect membrane retraction rates. Vps1 tagged with both GFP as well as superfolded GFP, and imaged by TIRF microscopy fails to colocalize with Abp1 (data not shown, personal communication with Andrea Picco). The debate concerning the involvement of Vps1 in membrane scission in yeast has been compounded by the possibility that the GFP tag at the Vps1 C-terminal could interfere with its localization to endocytic sites, or its interaction with the Rvs complex. 
	
		\subsubsection{Membrane scission is not dependent on yeast dynamin Vps1}
	\mbox{}\\
	In order to exclude the possibility of interference from the GFP tag, I investigated the role of Vps1 by studying coat and scission proteins in \textit{vps1Δ} cells. Movement of Sla1 and Rvs167 \textit{vps1Δ} is compared against that in WT cells. 
	
	\begin{figure}
	\centering
	\includegraphics[width=22cm,height=22cm,keepaspectratio]{figures/results_final/vps}
	\caption[Rvs localization in vps deletion]
	{A: Dot spots of WT, Vps1-GFP, and \textit{vps1$\Delta$} cells at 30C and 37C. 
		B: Kymographs of Sla1-GFP and Rvs167-GFP in WT and \textit{vps1$\Delta$}. Exposure 80ms.  
		C: Failure rate of membrane scission, measured from retractions of Sla1, or lack of movement. 
		D, E: Movement and normalized fluorescent intensity of Sla1-GFP in WT and \textit{vps1$\Delta$} cells. Time =0 (s) for WT Sla1 indicates scission time. \textit{vps1$\Delta$} Sla1  is shifted to move inwards at the same time as WT. 
		F, G: Movement and normalized fluorescent intensity of Rvs167-GFP in WT and \textit{vps1$\Delta$} strains. Time =0 (s) for WT Rvs167 indicates scission time. \textit{vps1$\Delta$} Rvs167 is shifted so that fluorescent intensity maxima is at time=0 (s).
		\label{fig4_vpsdel}}
	\end{figure}
	
	
	\vspace{5mm}
	\textit{vps1Δ }cells exhibit a growth defect at 37C, as has been reported11. Sla1-GFP accumulates at the plasma membrane, moves inwards, and disassembles like in WT, as seen in the kymograph in \ref{fig4_vpsdel}. In WT cells, Sla1 moves into the cytoplasm about 140nm before membrane scission occurs. If Vps1 was required for membrane scission, Sla1 would be expected to undergo delayed or failed scission. However, \textit{vps1Δ} does not increase the rate of membrane retraction. Inward movement of Sla1 is also not changed: it moves inward at the same rate, and to similar maximum of 140nm. Further, the averaged centroid of Rvs167 would not show the sharp jump into the cytoplasm if scission failed. If scission was delayed, the average lifetime of Rvs167 would increase. The inward movement of Rvs167, and its average lifetime, however, remains the same as in WT. I conclude that if Vps1 does localize to endocytic patches in S.cerevisiae, it is not involved in regulating membrane scission.  


	\subsection{BAR domains as recruiters of Synaptojanins} 
	\subparagraph{ Lipid hydrolysis and membrane scission}
	
	
	Another hypothesis has proposed that regulated lipid hydrolysis can cause vesicle scission19. Phosphatidylinositols (PIs) and their derivatives play important roles in many cellular processes including membrane trafficking and cell signalling. Conversion between lipid types is driven by kinases, lipases, and phosphatases and controlled throughout the membrane trafficking pathway. 

	\vspace{5mm}
	Phosphatidylinositol(4,5)-biphosphate (PI (4,5) P2) is an important lipid type found at the cell surface, and is enriched and depleted from endocytic sites at the plasma membrane in concert with the assembly and disassembly of the endocytic machinery. Synaptojanins form a subset of inositol polyphosphate 5-phosphatases that hydrolyze PI(4,5)P2 to PI(4)P by removing the phosphate at the 5’ position of the inositol ring, and play a role in CME and intracellular signalling, as well as in modulating the actin cytoskeleton14. Synaptojanins localize to endocytic sites, and in mammalian cells, disruption of Synaptojanin genes results in cellular accumulation of PI(4,5)P2 and coated vesicles at the plasma membrane, suggesting a role for lipid hydrolysis in releasing coat proteins from nascent vesicles. Syaptojanins contain an N-terminal homology domain with the cytoplasmic domain of the yeast SAC1 gene, that is implicated in lipid metabolism, actin morphology, and vesicle transport in the secretary pathway15. A central catalytic domain is followed by a proline-rich C-terminal regions that are the canonical interaction partners of SH3 domains: they are known to interact with actin binding proteins and BAR domain proteins, potentiating also a role in membrane invagination and scission. 
		
	\vspace{5mm}
	The yeast encodes three Synaptojanin-like proteins- Inp51, Inp52 and Inp53- that regulate phospholipid metabolism. Double deletion of Inp51 and Inp52 has been shown to increase the lifetime of endocytic proteins and produce aberrant membrane invaginations that could indicate scission failure and defective endocytosis, although uptake of extracellular membrane appears to proceed in spite of the morphological aberrations16,17. Deletion of Inp52 along with Rvs167 increases scission failure rate, supporting a possible role in membrane scission18. Loss of inp51 mutation shows a increase in bulk PIP2 level, although chages in PIP2 levels have not been reported for mutations of inp52, and are not measured locally at the endocytic sites19,20.
		

		\subsubsection{Synaptojanin-like Inp52 localizes to endocytic sites in the late scission stage}
		Of the three synaptojanin-like proteins in yeast, Inp51, Inp52, and Inp53, only Inp52-GFP localizes to cortical patches (Fig.\ref{fig_inp}A) that co-localize with Abp1 (not shown). Inp51-GFP does not show any cortical patches: GFP-tagging either affects localization of the protein, or localization is too to be detected. Spatial and temporal alignment of Inp52-GFP to the endocytic timeline shows that it is localized to endocytic sites at the final scission stage, along with Rvs167. Spatial alignment with the coat protein Sla1 as reference shows that Inp52-GFP localizes to the tip of the membrane tube, and remains on the vesicle for a few seconds post scission: spatial and temporal localization of Inp52 is consistent with influence on scission, and could indicate a role for Inp52 in vesicle uncoating. Inp51-GFP does not show any cortical patches: GFP-tagging either affects localization of the protein, or localization is too to be detected. Rvs in single deletion strains show localizations similar to WT, but double deletion strains consists of large patches of Rvs167 at the plasma membrane, as well as localized within the cytoplasm. 
		
			\begin{figure}[H]
			\centering
			\includegraphics[width=19cm,height=19cm,keepaspectratio]{figures/results_final/inp}
			\caption[Synaptojanin-like proteins in yeast]
			{A: Maximum intensity projections of time-lapse images of cells expressing GFP-tagged Synaptojanin-like proteins Inp51, Inp52, and Inp53. Exposure rate 80ms.
			B: Failure rate of membrane scission, measured by quantifying number of retractions of Sla1 after membrane begins to move inwards, or by total lack of movement in WT, \textit{rvs167$\Delta$}, \textit{inp51$\Delta$} and \textit{inp52$\Delta$} strains.
			C: Sla1-GFP in WT,  \textit{inp51$\Delta$}  and  \textit{inp52$\Delta$} strains show similar plasma membrane localization. Rvs167-GFP in WT, \textit{inp51$\Delta$}, \textit{inp52$\Delta$} and \textit{inp51$\Delta$}\textit{inp52$\Delta$} strains. 
			D: Inp52-GFP in endocytic timeline in WT cells. Time=0 (s) corresponds to scission time.
			All scale bars =2um.
			\label{fig_inp}}
			\end{figure}
				

		\subsubsection{Loss of yeast synaptojanins does not significantly affect coat and Rvs dynamics}
		In a model proposed by Liu et al, Synpatojanins and BAR proteins interact to regulate PI(4,5)P2 hydrolysis, which in turn drives membrane scission. Here, the Rvs scaffold on the membrane tube protects the underlying PIP2 from hydrolysis. Synaptojanin arrives at sites, and hydrolyses unprotected PIP2. This generates a boundary between BAR-protected PIP2 at the tube and PIP at the bud tip. The lipid boundary produces a line tension at the interphase that would generate enough force to pinch off a vesicle. 


		\vspace{5mm}
		In order to test this model, I quantified coat and scission mechanics in \textit{inp52$\Delta$ } cells.
		Deletion of Inp52 does not affect the speed of membrane invagination, or the maximum invagination length, as reported by the movement of the Sla1 centroid. Sla1-GFP patches are assembled and disassembled, as are Rvs167-GFP patches in \textit{inp52$\Delta$ } cells (\ref{}). \textit{All-xxxxxx} Sla1-GFP patches analysed (n=13 cells) move inwards. 72.9\% of Rvs167-GFP patches move into the cytoplasm (n=4 cells, 37 patches). Remaining patches are disassembled without inward movement. Vesicle scission in \textit{inp52$\Delta$ }occurs similar to WT, since the Rvs167 centroid moves inwards to approximately the same distance into the cytoplasm, indicating that the base of the vesicles are likely at the same position as in WT cells. Both Sla1 and Rvs167 centroids however, persist post-scission (arrowheads in figure) instead of disassembling immediately like in the WT. Since majority of the patches move inwards, and the increase in the lifetime of Rvs is post-scission, I find that the data is consistent with a role for Inp52 in vesicle uncoating, rather than a primary role in membrane scission, with the aberrations in plasma membrane morphology consequent of failure to recycle components, rather than scission. 


		\vspace{5mm}
		I also quantified the coat and scission dynamics \textit{inp51$\Delta$ }, since PiP2 levels have been found to increase in \textit{inp51$\Delta$ } cells. Loss of Inp51 does not affect Rvs167 or Sla1 centroid movement. All Sla1-GFP patches move inward (n=19 cells). 93\% of Rvs167-GFP patches move inward (n=3 cells, 44 patches), similar to WT. Assembly of Rvs167 in the \textit{inp51$\Delta$ }is slowed, the implication of this delay is not thus far clear. 

			\begin{figure}[H]
	\centering
	\includegraphics[width=19cm,height=19cm,keepaspectratio]{figures/results_final/inp_movement3}
	\caption[Synaptojanin deletion]
	{A: Movement of Sla1 in WT, Inp51del and Inp52del strains. 
		B: Median number of Rvs167 molecules recruited to endocytic sites in WT, \textit{inp52$\Delta$}, \textit{inp52$\Delta$ } and \textit{inp51$\Delta$inp52$\Delta$ } cells. P-values from two-sided z test,  * = p $\leq$ 0.05 , ** = p$\leq$ 0.01, *** = p $\leq$ 0.001.  
		C: Movement of Rvs167 in WT, \textit{inp51$\Delta$ } and \textit{inp52$\Delta$ } strains. 
		D: Normalized fluorescent intensity of averaged Rvs167 patches in WT, \textit{inp51$\Delta$ }, \textit{inp52$\Delta$ } cells.
		\label{fig_inpmov}}
\end{figure}

	
	\subsection{BAR domains as friction-inducing domains}
	
	Recent in-vitro experiments have proposed protein friction as a mechanism by which membrane scission could occur via BAR domain proteins in the absence of dynamin24. In this model, a BAR domain scaffold on a membrane tube forms a frictional barrier to lipid diffusion. Forces that pull on the membrane, such as those exerted by motor proteins like myosins or actin polymerization, increase the frictional force exerted by the scaffold on the underlying membrane tube, increasing membrane tension in the bare region of the tube caused by membrane thinning from the lack of lipid influx to this region. Eventually, membrane pores form in this portion of the tube, leading to breaking the tube, and formation of a vesicle. Such a friction-dependent membrane scission model would predict that if more BAR proteins are added to the membrane, frictional force would increase, and scission should occur faster, that is, at shorter invagination lengths than with fewer BAR proteins. Essentially, this model requires a friction-inducing BAR scaffold, and a force that pulls the membrane under it. In yeast CME, this combination is provided by the Rvs complex and polymerization of the actin network respectively. 
	

		\subsubsection{Membrane scission does not occur at shorter tube lengths when amount of Rvs is increased}

	To test whether protein friction could influence membrane scission in yeast, I duplicated the Rvs167 and Rvs161 genes as described in Huber et al.21. Gene duplication is performed in haploid cells to produce strains that have one (WT number) and two copies of both Rvs161 and Rvs167. 
		
				\vspace{5mm}
				\begin{figure}[h]
				\centering
				\includegraphics[width=13cm,height=13cm,keepaspectratio]{figures/results_final/rvs_haploid}
				\caption[Overexpression of the Rvs complex in haploid cells]
					{A. Movement of Sla1 in WT (1x RVS) and gene-duplicated Rvs (2x RVS) cells. 1x RVS Sla1 is aligned so that time=0 (s) is scission time. 2x RVS Sla1 is shifted to move inwards at the same time.
					B. Recruitment of Rvs167 to sites in WT (1x Rvs)  and gene-duplicated Rvs (2x RVS) cells.1x RVS is aligned so that time = 0 (s) is scission time. 2x RVS Sla1 is shifted to move inwards at the same time.
					C.  Movement of Rvs167-GFP in WT (1x RVS) and gene-duplicated Rvs (2x RVS) cells. 1x RVS Rvs167 is aligned so that time=0 (s) is scission time. 2x RVS Rvs167 is shifted to move inwards at the same time.
			 \label{fig_rvshaploid}}

				\end{figure}
		
		By quantifying the number of Rvs167 proteins as described in Picco et al., 12  in the haploid strains, I found that the average maximum number of Rvs molecules recruited to endocytic sites is increased, to 179.7328 +/- 10.1, from 113.505 +/- 5.2 in WT:  1.6x Rvs is recruited to endocytic sites in the duplicated strain. The averaged trajectory of Sla1-GFP, however does not change between the 4x and 2x copies of Rvs: like in the WT, Sla1 moves in to 140nm. The dynamics of Rvs, however is chaged: In Fig.\ref{fig5_rvshaploid} , the centroid movement of both Rvs167-GFP in WT and 2x RVS is aligned so that time=0 corresponds to the maxima of the fluorescent intensity, which corresponds to scission time. Molecule numbers of 2x Rvs increase at a faster rate than WT, and the disassembly is slowed by ~1 second. 
		
		In the corresponding Rvs movement trace, instead of the sharp jump seen in WT, there is a delay in the movement into the cytoplasm.



			\subsubsection{Scission timing occurs at a specific amount of Abp1, \\
			independent of the amount of Rvs recruited}
			The gene duplicated haploid strains are then crossed, so that diploid strains are generated that have 4x copies of the Rvs167 and Rvs161 genes, 2x copies each (WT number). Strains containing 1x copy of each Rvs is generated by crossing \textit{rvs167$\Delta$} strain with an \textit{rvs161$\Delta$} strain. I quantify Rvs recruitment, coat movement, and scission dynamics in these diploid strains, to study membrane scission as a function of Rvs copy number
		

	
			\vspace{5mm}
			In diploid cells, recruitment of Rvs is not directly proportionate to gene copy number: maximum number of Rvs recruited increases from 100.9 +/- 4.6 from in the 2x Rvs strain to 142.5, +/- 5.5 in the 4x strain. In the 1x Rvs strain, 80.2, +/- 4.7 molecules of Rvs are recruited before scission occurs. In order to determine whether this is a reflection on protein availability or if something else limits recruitment of Rvs, I quantified the cytoplasmic intensity of Rvs167-GFP in the respective strains by first producing z-stacks of time-lapse images of cells, and measuring the intensity within the cytoplasm (for details see METHODS). 	As can be seen in TABLE.X, the number of molecules recruited to endocytic sites scales with the amount of protein in the cytoplasm.  
			
			\vspace{5mm}
			Inward movement of the Rvs centroid is similar for the 4x, 2x and 1x Rvs: the jump inwards is about 80nm. In the 1x strain, however, the centroid disappears immediately after scission, suggesting that there is reduced Rvs at the base of the newly formed vesicle compared to the WT. Recruitment dynamics of all three are different: in the 4x Rvs strain, Rvs is recruited at a rate of 57 molecules/second, which is reduced to 27 molec/sec for the 2x and 19.07 molec/sec for the 1x strain. 
			
			\vspace{5mm}
			Sla1 centroid movement, meanwhile is the same in 4x and 2x Rvs strains. In the 1x Rvs strain, Sla1 movement is slightly decreased, suggesting that vesicle scission occurs at invagination lengths about 10nm shorter than that as WT when fewer Rvs molecules are recruited, but increasing Rvs recruitment does not affect membrane progression. 
			
\newpage
						\begin{figure}[H]
						\centering
						\includegraphics[width=22cm,height=22cm,keepaspectratio]{figures/results_final/protein_friction2}
						\caption[Dynamics of endocytosis in diploid strains with gene duplicated Rvs]
						{A. Movement of Sla1 in WT (1x RVS) and gene-duplicated Rvs (2x RVS) cells. 1x RVS Sla1 is aligned so that time=0 (s) is scission time. 2x RVS Sla1 is shifted to move inwards at the same time.
						B. Recruitment of Rvs167 to sites in WT (1x Rvs)  and gene-duplicated Rvs (2x RVS) cells.1x RVS is aligned so that time = 0 (s) is scission time. 2x RVS Sla1 is shifted to move inwards at the same time.
						C.  Movement of Rvs167-GFP in WT (1x RVS) and gene-duplicated Rvs (2x RVS) cells. 1x RVS Rvs167 is aligned so that time=0 (s) is scission time. 2x RVS Rvs167 is shifted to move inwards at the same time.
						\label{fig_rvsdiploid}}
						\end{figure}
				


	
	\subsection{BAR domains as membrane scaffolds}
	The capacity for BAR domains to oligomerize and tubulate liposomes has proposed membrane scaffolding as a possible function in vivo. As scaffolds they would impose their own curvature on the underlying membrane and stabilize this shape. There are some requirements for a protein complex to act as a scaffold:\\
	1. it must have a defined membrane interface\\
	2. it must have an intrinsic curvature\\
	3. it must present be rigid in structure, and\\
	4. membrane binding surface must be large enough to induce curvature\\
	
	
	 It has been shown that BAR domains can prevent membrane scission and allow formation of stable tubular structures. 
	
		\subsubsection{Localization of BAR domain alone prevents membrane scission}
		 In haploid cells, Rvs167 expressed without the SH3 domain is recruited in lower numbers than the full-length protein (Fig.1B right), although cellular expression of the protein is the same as the WT (not shown). Sla1 movement in rvs167ΔSH3 cells is reduced to close to rvs167Δ lengths (Fig.4 left). When the Rvs167 gene is duplicated without the SH3 domain, the number of rvs167ΔSH3 molecules recruited to endocytic sites are increased, and correspondingly, Sla1 movement increases from the rvs167Δ lengths towards the WT lengths (Fig.4): shallow invaginations of the rvs167Δ can be rescued by recruiting only BAR domains of Rvs167.
		 
		\begin{figure}[H]
		\centering
		\includegraphics[width=25cm,height=25 cm,keepaspectratio]{figures/results_final/scaffolding_overlaid2}
		\caption [Progression of invagination with increasing BAR recruitment]
		{A:Averaged centroid movement of Sla1 in strains expressing varying gene copy numbers of Rvs complex, and \textit{rvs167$\Delta$}. 1x Rvs=WT expression, 2x = Rvs167 and Rvs161 genes are duplicated. 1xBAR=Rvs167 without the SH3 domain, 2xBAR=Rvs167 without the SH3 domain is duplicated. Averages are aligned in time so that time=0 (s) for WT Sla1-GFP corresponds to scission time. Other averages have been shifted to move inwards at the same time as the WT.
		B: Averaged centroid movement of Rvs167-GFP in 1x RVS, 2xRVS, 1xBAR, 2XBAR strains.
		C:  }
		\label{fig_scaffold}
		\end{figure}


	\subsection{Membrane dynmaics with reduced membrane tension}
	
	Yeast cells are under high turgor pressure by virtue of the rigid cell wall, expected to be several orders of magnitude higher than cultured mammalian cells. Forces from actin polymerization are hence necessary to overcome this resistance to membrane bending. Membrane tension, which also contributes to the resistance to bending, has a large influence on the forces required to invaginate the membrabe. Osmotic pressure, and therefore, membrane tension can be controlled 
	
	\begin{figure}[H]
	\centering
	\includegraphics[width=10cm,height=10 cm,keepaspectratio]{figures/results_final/sorbitol}
	\end{figure}



